cvm/docker-compose.yml

```diff
@@ -1,32 +1,55 @@
 services:
-  vllm:
+  vllm_service:
     image: vllm/vllm-openai:latest
-    container_name: vllm
-    environment:
-      - NVIDIA_VISIBLE_DEVICES=all
+    container_name: vllm_container
+    env_file:
+        - .env_prod
+    gpus: all
+    # runtime: nvidia
+    ports:
+      - "${PORT_PROVIDER}:${PORT_PROVIDER}"  # host:container port mapping
```

Comment:
why is VLLM being exposed to the outside? It should only be accessible internally